lang/java/ipc/src/test/java/org/apache/avro/ipc/TestNettyServerWithCallbacks.java

```diff
@@ -21,13 +21,7 @@
 import java.io.IOException;
 import java.net.InetSocketAddress;
 import java.nio.ByteBuffer;
-import java.util.concurrent.CountDownLatch;
-import java.util.concurrent.ExecutionException;
-import java.util.concurrent.ExecutorService;
-import java.util.concurrent.Executors;
-import java.util.concurrent.Semaphore;
-import java.util.concurrent.TimeUnit;
-import java.util.concurrent.TimeoutException;
+import java.util.concurrent.*;
```

Comment:
I sort of preferred the imports how they were before. It's a bit inconsistent now because the lines below this one don't use a wildcard import.